<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>948</width>
    <height>648</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Audiobook Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/SVG/svg/icon/AudiobookEditorLogo@16.svg</normaloff>:/SVG/svg/icon/AudiobookEditorLogo@16.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>436</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="font">
     <font>
      <family>Noto Sans TC</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="font">
      <font>
       <family>Noto Sans TC</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Select your language</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <property name="separatorsCollapsible">
      <bool>false</bool>
     </property>
     <property name="toolTipsVisible">
      <bool>true</bool>
     </property>
     <addaction name="action_EN"/>
     <addaction name="action_TC"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="menu_Language"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="action_Preview"/>
    <addaction name="action_Pack"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="action_PyQt5"/>
    <addaction name="separator"/>
    <addaction name="action_Version"/>
    <addaction name="action_Thanks"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Project"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Help"/>
   <addaction name="action_Pack"/>
   <addaction name="action_Validate"/>
   <addaction name="action_Preview"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Left">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>76</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>524287</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #404040;
border: 0px;

QTabBar::tab {
	left: 0
}
</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Left DockWidget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans TC</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::tab-bar{
height: 32px;
width: 166px;
font-size: 13px;
line-height: 18px;
}

QTabBar::tab {
	background-color : #404040;
}

QTabBar::tab::unselected { 
	color: rgba(255, 255, 255, 0.4);
	height: 32px;
}

QTabBar::tab::selected { 
	color: #FFFFFF;
	height: 32px;
}

QTabWidget::pane { border: 0; }

margin: 0px 0px;</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_Metadata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;
</string>
        </property>
        <attribute name="title">
         <string>Metadata</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_TOC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 0px;

/*
QListWidget::item { border: 2px solid #000000 }
*/</string>
        </property>
        <attribute name="title">
         <string>TOC</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_CoverPreviewWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>216</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>216</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #404040;
border: 0px;
</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Cover Preview Widget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>232</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>232</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_SupplementalList">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>39</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>524287</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #404040;
border: 0px;
margin: 0px;</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Supplemental List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;

padding: 0px;

margin: 0px;</string>
    </property>
   </widget>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>&amp;Qt</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_PyQt5">
   <property name="text">
    <string>&amp;PyQt5</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Preview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/toolbar-preview.svg</normaloff>:/SVG/svg/icon/toolbar-preview.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Preview</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Validate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/toolbar-validate.svg</normaloff>:/SVG/svg/icon/toolbar-validate.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Validate</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Pack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/toolbar-package.svg</normaloff>:/SVG/svg/icon/toolbar-package.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Package</string>
   </property>
   <property name="toolTip">
    <string>Pack</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Help">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/toolbar-about.svg</normaloff>:/SVG/svg/icon/toolbar-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Version">
   <property name="text">
    <string>&amp;Version</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Thanks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Thanks</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_EN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true" extracomment="EN">English</string>
   </property>
   <property name="toolTip">
    <string notr="true">English</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_TC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true" extracomment="TC">繁體中文</string>
   </property>
   <property name="toolTip">
    <string notr="true">繁體中文</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
